src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -78,6 +78,12 @@ internal static class FeatureOnOffOptions
 
         public static readonly PerLanguageOption<bool> StreamingGoToImplementation = new PerLanguageOption<bool>(
             nameof(FeatureOnOffOptions), nameof(StreamingGoToImplementation), defaultValue: true);
+
+        private const string LocalRegistryPath = @"Roslyn\Features\";
+
+        public static readonly Option<bool> NavigateToDecompiledSources = new Option<bool>(
+            nameof(FeatureOnOffOptions), nameof(NavigateToDecompiledSources), defaultValue: false,
+            storageLocations: new LocalUserProfileStorageLocation(LocalRegistryPath + nameof(NavigateToDecompiledSources)));
```

Comment:
By making this be LocalUserProfileStorageLocation we're implicitly saying this isn't roaming. Is that intentional? I'd imagine we want this to roam between machines.

Reply:
Or is the assumption here that somebody in different legal contexts (contractor who uses multiple machines) needs different options? Not saying this is wrong, just want to make sure this was intentional vs. something that fell out by copypasta.

Reply:
➡️ I used local user storage as the safe handling for legal considerations until we validate that roaming the option is not a problem. I anticipate making this a roaming option at the same time we are comfortable enabling it by default.

Reply:
:memo: On second look, this is just the feature flag. I'm going to change this to roaming, which allows us to see the number of people who are using the new feature. The legal disclaimer is separate and the acceptance of that will *not* roam (see #23479).

Reply:
➡️ Fixed in 24ecfb8ccdff844b9e1f3dcab53a2c38f854eaa6